_config.yml

```diff
@@ -1,10 +1,10 @@
 title: OpenLattice Help
 email: support@openlattice.com
-description: > # Welcome to Loom. These are the support resources to help you use the Loom platform.
+description: > # Welcome to OpenLattice. These are the support resources to help you use the OpenLattice platform.
 # baseurl: "" # the subpath of your site, e.g. /blog
 # url: "" # the base hostname & protocol for your site, e.g. http://example.com
 logo: openlattice_logo.png
-loom: https://thedataloom.com/
+loom: https://openlattice.com
```

Comment:
Change `loom` to either `openlattice` or `homepage` or `openlattice_homepage`. Additionally, you'll have to update other places in the project that refers to `site.loom`.

_guides/signups.md
_includes/header.html